src/Api/Controllers/ServiceAccountsController.cs

```diff
@@ -9,37 +10,52 @@
 namespace Bit.Api.Controllers;
 
 [SecretsManager]
+[Route("service-accounts")]
 public class ServiceAccountsController : Controller
 {
     private readonly IServiceAccountRepository _serviceAccountRepository;
+    private readonly IApiKeyRepository _apiKeyRepository;
     private readonly ICreateServiceAccountCommand _createServiceAccountCommand;
     private readonly IUpdateServiceAccountCommand _updateServiceAccountCommand;
 
-    public ServiceAccountsController(IServiceAccountRepository serviceAccountRepository, ICreateServiceAccountCommand createServiceAccountCommand, IUpdateServiceAccountCommand updateServiceAccountCommand)
+    public ServiceAccountsController(
+        IServiceAccountRepository serviceAccountRepository,
+        IApiKeyRepository apiKeyRepository, ICreateServiceAccountCommand createServiceAccountCommand,
+        IUpdateServiceAccountCommand updateServiceAccountCommand)
     {
         _serviceAccountRepository = serviceAccountRepository;
+        _apiKeyRepository = apiKeyRepository;
         _createServiceAccountCommand = createServiceAccountCommand;
         _updateServiceAccountCommand = updateServiceAccountCommand;
     }
 
-    [HttpGet("organizations/{organizationId}/service-accounts")]
+    [HttpGet("/organizations/{organizationId}/service-accounts")]
     public async Task<ListResponseModel<ServiceAccountResponseModel>> GetServiceAccountsByOrganizationAsync([FromRoute] Guid organizationId)
     {
         var serviceAccounts = await _serviceAccountRepository.GetManyByOrganizationIdAsync(organizationId);
         var responses = serviceAccounts.Select(serviceAccount => new ServiceAccountResponseModel(serviceAccount));
         return new ListResponseModel<ServiceAccountResponseModel>(responses);
     }
-    [HttpPost("organizations/{organizationId}/service-accounts")]
+
+    [HttpPost("/organizations/{organizationId}/service-accounts")]
     public async Task<ServiceAccountResponseModel> CreateServiceAccountAsync([FromRoute] Guid organizationId, [FromBody] ServiceAccountCreateRequestModel createRequest)
     {
         var result = await _createServiceAccountCommand.CreateAsync(createRequest.ToServiceAccount(organizationId));
         return new ServiceAccountResponseModel(result);
     }
 
-    [HttpPut("service-accounts/{id}")]
+    [HttpPut("{id}")]
     public async Task<ServiceAccountResponseModel> UpdateServiceAccountAsync([FromRoute] Guid id, [FromBody] ServiceAccountUpdateRequestModel updateRequest)
     {
         var result = await _updateServiceAccountCommand.UpdateAsync(updateRequest.ToServiceAccount(id));
         return new ServiceAccountResponseModel(result);
     }
+
+    [HttpGet("{id}/access-tokens")]
```

Comment:
Do we need a "/" here, since on the other methods we have /organizations?

[HttpGet("/{id}/access-tokens")]